oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java

```diff
@@ -168,6 +161,46 @@ public void in(MetaInAlarm meta, Metrics metrics) {
         }
     }
 
+    private boolean validate(String target, List<String> includeList, List<String> excludeList,
```

Comment:
You are reusing this, please put comments on the method.

Reply:
done